room-kit/src/main/java/live/hms/roomkit/ui/meeting/MeetingFragment.kt

```diff
@@ -332,7 +332,14 @@ class MeetingFragment : Fragment() {
         requireActivity().finish()
     }
 
-    private fun updateRecordingViews(state: HMSRecordingState) {
+    private fun updateRecordingViews(state: HMSRecordingState , isRecordingIconsEnabled : Boolean?) {
```

Comment:
it might be better if changing "isRecordingIconsEnabled" name to "shouldShow" or something like that, 
also else the visibility variable introduction using will better than this approach